library/core/tests/net/parser.rs

```diff
@@ -1,6 +1,6 @@
 // FIXME: These tests are all excellent candidates for AFL fuzz testing
-use crate::net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr, SocketAddrV4, SocketAddrV6};
-use crate::str::FromStr;
+use core::net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr, SocketAddrV4, SocketAddrV6};
+use core::str::FromStr;
```

Comment:
I think these tests are dead code now. Unit tests are not run for libcore:

https://github.com/rust-lang/rust/blob/1291216ac986ccc1432bf0bf2db9545e5251dd24/library/core/Cargo.toml#L13-L15